drivers/filesystems/cdfs_new/fsctrl.c

```diff
@@ -906,7 +1009,7 @@ Return Value:
                 //  Since we're always last in the mount order,  we won't be preventing
                 //  any other FS from trying to mount the data track.  However if the 
                 //  data track was at the start of the disc,  then we abort,  to avoid
-                //  having to filter it from our synthesized directory listing later.  We
+                //  having to filter it from our synthesised directory listing later.  We
```

Comment:
In this specific case, the spelling change is british vs american.. :P
I chose not to "carry over" the spelling fixes we had before, since they are unnecessary diff noise. for the person syncing the driver in the future, but if someone sends them upstream then we can sync it afterward?